Add Organizations to Readme

alexvonme opened the pull request:
To close #19

ccoVeille commented on README.md:
```diff
@@ -9,7 +9,7 @@ The repository to find popular Vale rules that are:
 [Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important advantage of the software is its capability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/).
 
-Many organizations that use this capability with Vale also publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
+Many organizations that use Vale, sincluding [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon  ...
```
```suggestion
Many organizations that use Vale, including [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
```

ccoVeille replied:
Also, we could mention the official Vale packages.

https://vale.sh/hub/

Not necessarily in the same sentences about organizations.

ccoVeille replied:
For records, we could apply to be listed on Vale hub when the project will be more mature

alexvonme replied:
I absolutely agree with this one, in a sense that should be a goal of the project.

ccoVeille replied:
That's what I imagined when I thought about the project. I'm glad we are aligned

ccoVeille commented on README.md:
```diff
@@ -9,7 +9,7 @@ The repository to find popular Vale rules that are:
 [Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important advantage of the software is its capability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/).
 
-Many organizations that use this capability with Vale also publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
+Many organizations that use Vale, sincluding [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon  ...
```
```suggestion
Many organizations that use Vale, sincluding [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results. jargonLint will focus on non-ambiguous rules. This way jargonLint could be used in addition of these organizations Vale rules
```

To be rephrase maybe.

It can be addressed later than this PR

alexvonme replied:
Yeah, I've already thought of this, but we should address this in a later PR, preferably when we have and can test for 5+ rules.

ccoVeille replied:
Should we create an issue for it?

Or a todo in a discussion

ccoVeille replied:
No need, I just found put you wrote about it in the new version of the README

alexvonme commented:
I edited around the readme, you may benefit from reading it over like a new text from my branch, [here](https://github.com/alexvonme/jargonLint/blob/readme/README.md).

ccoVeille commented on README.md:
```diff
@@ -7,15 +7,15 @@ The repository to find popular Vale rules that are:
+[Vale](https://vale.sh/) is an open source documentation software, accessible both from the [command-line](https://vale.sh/docs/vale-cli/installation/) or through [integrations](https://vale.sh/docs/integrations/guide/). An important feature of this tool is the ability to support custom [styles](https://vale.sh/docs/topics/styles/) and [lint actions](https://vale.sh/docs/topics/actions/). Vale de ...
 
-Many organizations that use this capability with Vale also publish their configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many documentation writers have re-iterated upon similar rules delivering the same results.
+Furthermore, organizations including [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their Vale configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many technical writers have re-iterated upon similar ...
```
```suggestion
## Vale ecosystem and its limits

Furthermore, organizations including [Red Hat](https://github.com/redhat-documentation/vale-at-red-hat) and [Splunk](https://github.com/splunk/vale-splunk-style-guide), often publish their Vale configurations with permissive licenses. The repositories with these rulesets can be very useful to learn and adapt from. However, they also reveal that many technical writers have re-iterated upon similar rules, aiming to deliver the same results.
```

I think we need a section here. The title of the section can we rewritten.

I suggest this because this way, we would have a :

- What it's about
- the problems
- our solution

With current version:
- (many/lazy/fast-reading) people won't read the last sentences of vale section, while it's the most important.

And these last sentences are somehow opinionated, while the rest are descriptive, and factual.

alexvonme replied:
Good idea, but I'd prefer it to be a subsection instead. I'll update the README shortly, please review.

alexvonme commented:
I changed the ``Our contributions`` part. It was in past tense which I converted to present tense and converted ``applicability`` to ``application``. The former is probably better but also a much more difficult word.

ccoVeille commented:
Feel free to merge whenever you want

alexvonme commented:
Slightly changed the code of conduct title, merging now.
